src/Redist/Microsoft.ML.DnnImageFeaturizer.ModelRedist/Microsoft.ML.DnnImageFeaturizer.ModelRedist.proj

```diff
@@ -53,6 +53,46 @@
     DestinationDir="$(ModelPlacementDir)\ResNet101Onnx" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ModelFileCopy Include="$(ModelPlacementDir)\AlexNetPrepOnnx\AlexNetPreprocess.onnx"
+    TargetPath="$(PackageAssetsPath)\Microsoft.ML.DnnImageFeaturizer.AlexNet\tools\DnnImageModels\AlexNetPrepOnnx\AlexNetPreprocess.onnx" />
+  </ItemGroup>
```

Comment:
Out of curiosity, why we download them in one place and then copying them to another?
Why we can't just have them in one place? What's the point of copy?